test/test_autograd.py

```diff
@@ -2248,6 +2248,13 @@ def gen_matrices(p):
         gradgradcheck(torch.chain_matmul, gen_matrices([3, 5, 2, 6]))
         gradgradcheck(torch.chain_matmul, gen_matrices([6, 2, 4, 8, 10]))
 
+    def test_mvlgamma(self):
+        def gen_inputs(p, shape):
+            return torch.empty(*shape).double().uniform_(0.5 * (p - 1), p).requires_grad_()
```

Comment:
does it make sense to add this to https://github.com/pytorch/pytorch/blob/00a4c8d41cf8a7504a4300b3fc741d8a4ccc5795/test/common_methods_invocations.py#L100-L108 ?

Reply:
Sure, I think that is a better option. Thank you.